sources/hachiko/src/config.rs

```diff
@@ -0,0 +1,58 @@
+use crate::error::{self, Result};
+use serde::Deserialize;
+use snafu::ResultExt;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+#[derive(Debug, Deserialize)]
+pub(crate) struct Config {
+    pub(crate) should_wait: bool,
+    pub(crate) marker_path: String,
+}
+
+impl Config {
+    /// Parses configuration file at passed in path and returns struct containing settings value.
+    pub(crate) fn from_file<P: AsRef<Path>>(path: P) -> Result<Self> {
+        let path = path.as_ref();
+        let string_parse = fs::read_to_string(path).context(error::ConfigReadSnafu { path })?;
+        let config: Config =
+            toml::from_str(&string_parse).context(error::ConfigParseSnafu { path })?;
+        Ok(config)
+    }
+}
+
+#[cfg(test)]
+mod configTest {
+    use crate::config::Config;
+    use tempfile::TempDir;
+
+    // Example config file where user sets autoscaling.should-wait to false.
+    const false_should_wait: &str = r#"
+    should_wait = false
+    marker_path = "/var/lib/bottlerocket/hachiko.ran"
+    "#;
+
+    // Example config file where user sets autoscaling.should-wait to true.
+    const true_should_wait: &str = r#"
```

Comment:
`Const` variable needs upper case
```suggestion
    const TRUE_SHOULD_WAIT: &str = r#"
```